cmd/bootstrap/cmd/keys.go

```diff
@@ -90,13 +92,23 @@ func assembleNodeInfo(nodeConfig model.NodeConfig, networkKey, stakingKey crypto
 	return nodeInfo
 }
 
-func assembleNodeMachineAccountPriv(machineKey crypto.PrivateKey) model.NodeMachineAccountPriv {
+func assembleNodeMachineAccountInfo(accountAddress string, keyIndex uint, machineKey crypto.PrivateKey) model.NodeMachineAccountInfo {
+	log.Debug().Str("machineAccountPubKey", pubKeyToString(machineKey.PublicKey())).Msg("encoded public machine account key")
+	machineNodeInfo := model.NodeMachineAccountInfo{
+		EncodedPrivateKey: machineKey.Encode(),
+		KeyIndex:          keyIndex,
+		SigningAlgorithm:  sdkcrypto.ECDSA_P256,
+		HashAlgorithm:     sdkcrypto.SHA3_256,
```

Comment:
Let's hard-code the key index as well - it will always be 0 for the Flow Port flow we're supporting with this tool.